packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -771,14 +771,11 @@ function runActTests(label, render, unmount, rerender) {
           expect(document.querySelector('[data-test-id=spinner]')).toBeNull();
 
           // trigger a suspendy update with a delay
-          React.unstable_withSuspenseConfig(
-            () => {
-              act(() => {
-                rerender(<App suspend={true} />);
-              });
-            },
-            {timeout: 5000},
-          );
+          React.unstable_startTransition(() => {
+            act(() => {
```

Comment:
Should this work outside of startTransition?

Reply:
`act` should go on the outside but it's unrelated to this PR